app/components/TrackRow/index.js

```diff
@@ -1,18 +1,30 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 import PropTypes from 'prop-types';
 import _ from 'lodash';
 import numeral from 'numeral';
 import { Icon } from 'semantic-ui-react';
 
 import artPlaceholder from '../../../resources/media/art_placeholder.png';
 
+import * as QueueActions from '../../actions/queue';
+
 import TrackPopupContainer from '../../containers/TrackPopupContainer';
 import { formatDuration } from '../../utils';
 
 import styles from './styles.scss';
 
 class TrackRow extends React.Component {
-  getThumbnail(track) {
+  constructor () {
+    super();
+    this.state = { doubleClick: false };
```

Comment:
I think we won't need this in this component anymore, and it can be converted to a stateless one.